website/content/docs/connect/distributed-tracing.mdx

```diff
@@ -0,0 +1,236 @@
+---
+layout: docs
+page_title: Distributed Tracing
+description: >-
+  Distributed tracing is a way to track and correlate requests across microservices.
+---
+
+# Distributed Tracing
+
+Distributed tracing is a way to track and correlate requests across microservices. Distributed tracing must first
+be implemented in each application, it can't be added by Consul. Once implemented in your applications, adding
+distributed tracing to Consul will add the sidecar proxies as spans in the request path.
+
+## Application Changes
+
+Consul alone can't implement distributed tracing for your applications. Each application must propagate the required
+headers. Typically this is done using a tracing library.
```

Comment:
@lkysow can you list some libraries you would recommend? That way can help the reader by stating `Typically this is done using a tracing library such as ......`